presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixConfig.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.rubix;
+
+import com.qubole.rubix.spi.CacheConfig;
+import io.airlift.configuration.Config;
+
+public class RubixConfig
+{
+    private boolean cacheEnabled;
+    private boolean parallelWarmupEnabled = true;
+    private String cacheLocation = "/tmp";
+    private int bookKeeperServerPort = CacheConfig.DEFAULT_BOOKKEEPER_SERVER_PORT;
+    private int dataTransferServerPort = CacheConfig.DEFAULT_DATA_TRANSFER_SERVER_PORT;
+
+    public boolean isCacheEnabled()
+    {
+        return cacheEnabled;
+    }
+
+    @Config("hive.cache.enabled")
+    public RubixConfig setCacheEnabled(boolean value)
+    {
+        this.cacheEnabled = value;
+        return this;
+    }
+
+    public boolean isParallelWarmupEnabled()
+    {
+        return parallelWarmupEnabled;
+    }
+
+    @Config("hive.cache.parallel-warmup-enabled")
+    public RubixConfig setParallelWarmupEnabled(boolean value)
+    {
+        this.parallelWarmupEnabled = value;
+        return this;
+    }
+
+    public String getCacheLocation()
+    {
+        return cacheLocation;
+    }
+
+    @Config("hive.cache.cache.location")
```

Comment:
This looks a bit strange. How about `hive.cache.location`

Reply:
changing this